test/fixtures/local-context.yml

```diff
@@ -31,7 +31,13 @@ sources:
       - go.*
     excludes:
       - go.sum
-
+  src6:
```

Comment:
Are these sources actually being used?

Reply:
thanks, not checking this as part of the build stage